article/app/views/package.scala

```diff
@@ -1,10 +1,23 @@
 package views
 
 import common.Edition
+import layout.ContentWidths.MainMedia
 import model.Article
 import play.api.mvc.RequestHeader
 import views.support._
 
+object Widths {
+
+  def apply(isShowcase: Boolean, isFeature: Boolean): layout.WidthsByBreakpoint = {
```

Comment:
it could - I went for booleans because I didn't think an article as a whole is relevant to the main image widths, however I could go either way on it - I'll update the name of Widths to be more informative at the same time.  thanks Rich.